lagunita/lms/static/sass/discussion/lms-discussion-main.scss

```diff
@@ -1,3 +1,10 @@
 @import 'lms/static/sass/discussion/lms-discussion-main';
 
 @import '../overrides';
+
+.discussion-user-profile-board {
```

Comment:
You may want to start (maybe wrap the whole thing) your DOM selection with an ID. I'll show you tomorrow.

Reply:
When you get the chance, can you please show me what you meant by this?

Reply:
@caesar2164 Why though? Front-end development should avoid using IDs as a general rule, unless explicitly needed.

It often causes problems.
Everyone always says "there will never be more than one"... but that doesn't always hold.. Sometimes a second gets added to the page because we're cloning the element via jQuery, etc..

Why do you suggest this change here?
Why do we need an explicitly unique identifier?

Reply:
Actually, I see the ID is existing and we aren't creating a new ID for element, but my general objection remains...

Why make this more brittle by also requiring the ID to be present?
edX could remove that at any time. Why rely on it if we don't have to?

Reply:
Because we're overriding edx-platform style. Using the ID ensures our style win by being more specific and not needing `!important`, and not breaking as much through merges...